src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                      image={getImage(data.rhino)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/staking/">Start staking</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Run a node"
+                description="Run a node to play a crucial part in the Ethereum network."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.ethereumInside)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/run-a-node/">Run a node</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[4].id}>{tocItems[4].title}</h2>
+            <p>
+              For users most interested in the technical part of the Ethereum
+              network.
+            </p>
+            <CardGrid>
+              <Card
+                title="Energy Consumption"
+                description="How much energy does it cost to run Ethereum network?"
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.hackathon)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/energy-consumption/">
+                    Is Ethereum green?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Ethereum upgrades"
+                description="How are we going to make Ethereum more scalable, secure, and sustainable?"
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.merge)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/upgrades/">Read more</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Ethereum Whitepaper"
+                description="The original Ethereum proposal written by Vitalik Buterin in 2014."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.financeTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/whitepaper/">Read whitepaper</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              More about the Ethereum protocol
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/developers/">
+                Intro to Ethereum for Developers
```

Comment:
1. Lowercase d in developers
2. Is this an intro to Ethereum or intro to developing on Ethereum?
3. Should this be in the protocol section?

```suggestion
                Intro to Ethereum for developers
```

```suggestion
                Ethereum for developers
```

```suggestion
```